homework_14_140123/src/MethodsImplementation.java

```diff
@@ -0,0 +1,409 @@
+            int i;
+            do {
+                i = bufferedReader.read();
+                if (i != -1) {
+                    line += (char) i;
+                }
+            } while (i != -1);
+            System.out.println(line);
+            bufferedWriter2 = new BufferedWriter(new FileWriter(fileCopy));
+            String newline = line.replaceAll("[aeiouAEIOU]", "");
+            bufferedWriter2.write(newline);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
+                bufferedReader.close();
+                bufferedWriter2.close();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+    }
+
+
+    //Task8
+//    Реализовать метод который сравнивает содержание двух файлов,
+//    если они идентичны - вывести на экран, если нет - вывести на екран
+    public static void compareTextFiles() {
+        File file1 = new File("firstTestFile.txt");
+        File file2 = new File("secondTestFile.txt");
+        File file3 = new File("thirdTestFile.txt");
+        BufferedReader bufferedReader1 = null;
+        BufferedReader bufferedReader2 = null;
+        BufferedReader bufferedReader3 = null;
+        try {
+            bufferedReader1 = new BufferedReader(new FileReader(file1));
+            bufferedReader2 = new BufferedReader(new FileReader(file2));
+            bufferedReader3 = new BufferedReader(new FileReader(file3));
+            int i;
+            int j;
+            int k;
+            String line1 = "";
+            String line2 = "";
+            String line3 = "";
+            do {
+                i = bufferedReader1.read();
+                if (i != -1) {
+                    line1 += (char) i;
+                }
+            } while (i != -1);
+            do {
+                j = bufferedReader2.read();
+                if (j != -1) {
+                    line2 += (char) j;
+                }
+            } while (j != -1);
+            do {
+                k = bufferedReader3.read();
+                if (k != -1) {
+                    line3 += (char) k;
+                }
+            } while (k != -1);
+            if (line1.equals(line2)) {
+                System.out.println("The contents of file 'firstTestFile' and file 'secondTestFile' are the same");
+            } else {
+                System.out.println("The contents of file 'firstTestFile' and file 'secondTestFile' do not match");
+            }
+            if (line1.equals(line3)) {
+                System.out.println("The contents of file 'firstTestFile' and file 'thirdTestFile' are the same");
+            } else {
+                System.out.println("The contents of file 'firstTestFile' and file 'thirdTestFile' do not match");
+            }
+        } catch (IOException e) {
+            System.out.println("File not found");
+        } finally {
+            try {
```

Comment:
you don't need try block here, just close stream

Reply:
fixed. using try with resources.